File: frontend/src/app/models/power.ts
export class Power {
  public name: string;
  public mainTrait: string;
  public strength: number;
  public element: string;
  public details: string;
  public id?: number;

  constructor(name: string, mainTrait: string, strength: number, element: string, details: string) {
    this.name = name;
    this.mainTrait = mainTrait;
    this.strength = strength;
    this.element = element;
    this.details = details;
  }
}

File: backend/Fabrit.Heroes.Data.Entities/Power.cs
using Fabrit.Heroes.Infrastructure.Common.Data;

namespace Fabrit.Heroes.Data.Entities
{
    public class Power : IDataEntity
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Element { get; set; }
        public string Details { get; set; }
        public int Strength { get; set; }
        public string MainTrait { get; set; }
    }
}